### ISSUE TITLE
Artifacts when screen sharing application on Discord

### ISSUE THREAD

**Issue by erazemk** [stax76/mpv.net#225]
**Describe the bug**
When screen sharing an mpv.net window in the Discord desktop client (doesn't seem to happen in the browser) there are artifacts, such as a black screen at random times.
I tested with VLC, which didn't have any artifacts, so it's most likely an issue with mpv.net, not Discord.

**To Reproduce**
1. Play any video in mpv.net
2. In a local Discord client share the mpv.net window (haven't tested sharing the whole screen)
3. Users watching the stream should start seeing artifacts at random times (moving the mouse might increase occurrences)

**Expected behavior**
No artifacts when screen sharing.

**Screenshots**
The artifacts occur randomly and for a short time, so I can't reliably take screenshots.

**Comment by shirt-dev** [comment]
This occurs on most players including MPC-BE and regular MPV. A bug should be opened with discord or MPV themselves.

**Comment by erazemk** [comment]
> This occurs on most players including MPC-BE and regular MPV. A bug should be opened with discord or MPV themselves.

Ok, I'll open an issue with MPV. Should I close this one or keep it open?

**Comment by shirt-dev** [comment]
To be honest this is an issue with Discord, not the players.

**Comment by erazemk** [comment]
You sure? Because it doesn't happen with VLC. I mean it might still be a problem with Discord, but VLC still does something different than mpv, otherwise it would have the same problem.

**Comment by shirt-dev** [comment]
Yeah I'm pretty sure that they have some special code with VLC that breaks other players.
